include/sdcard/sdcard.h

```diff
@@ -29,15 +29,34 @@
 
 CPP_GUARD_BEGIN
 
+#define SD_EXTRA_STACK 128
+
+FRESULT sd_open( FIL* file, const TCHAR* path, BYTE mode );
+FRESULT sd_close( FIL* file );
+FRESULT sd_write( FIL* file, const void* buf, UINT btr, UINT* br );
+FRESULT sd_read( FIL* file, void* buff, UINT btr, UINT* br );
+FRESULT sd_lseek( FIL* file, DWORD ofs );
+FRESULT sd_sync( FIL* file );
+FRESULT sd_truncate( FIL* file );
+
+TCHAR*  sd_gets( TCHAR* buff, int len, FIL* fp );
+DWORD   sd_size( FIL* file );
+int	sd_puts (const TCHAR* str, FIL* cp);	
```

Comment:
It might be beneficial to inline the functions above.  Should to a code size test to compare.